public/styles/header.css

```diff
@@ -1,21 +1,24 @@
 header {
   display: flex;
   flex-direction: column;
-  background-color: #989e78;
+  /* background-color: #989e78; */
```

Comment:
Not necessary, but don't sleep on [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/var()), which are super useful for centralizing the color choices of your app, and let you reference them across your stylesheet. Then, if you want to slightly change a color or replace altogether, you can modify in one place and have all of the references update easily!